pennylane/capture/base_interpreter.py

```diff
@@ -675,6 +676,10 @@ def flattened_cond(self, *invals, jaxpr_branches, consts_slices, args_slice):
 
     for pred, jaxpr, const_slice in zip(conditions, jaxpr_branches, consts_slices):
         consts = invals[const_slice]
+        if math.is_abstract(pred):
+            raise NotImplementedError(
+                f"{self}  does not yet support jitting cond with abstract conditions."
```

Comment:
Looks great! Just one question. Do we need two spaces between `{self}` and "does"?

Reply:
```suggestion
                f"{self} does not yet support jitting cond with abstract conditions."
```
Thanks for catching that.